stockmetrics/metricsDescriptions/metricsDescriptions/HalsteadProgramLevel.html

```diff
@@ -0,0 +1,10 @@
+<html>
+<body>
+<table><tr><td valign="top" height="150">
+    <font face="verdana" size="-1">
+        Halstead Program Level defines a ratio between the Potential Volume of the algorithm and its actual volume.
```

Comment:
This description does not really explain anything :)
I guess some references to the papers defining these metrics, Wikipedia or whatever could help.